bitnami/airflow/templates/role.yaml

```diff
@@ -0,0 +1,27 @@
+{{- if .Values.rbac.create -}}
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: Role
+metadata:
+  name: {{ template "airflow.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels: {{- include "airflow.labels" . | nindent 4 }}
+    {{- if .Values.commonLabels }}
```

Comment:
You have added the parameter `commonLabels` to the new templates but it is not present in `values.yaml`. This is a parameter that we are introducing in our charts and I agree it should also be in this one. 

It would be great if you could implement this parameter in all the other templates. To keep using the `nginx-ingress-controller` as an example, you can use #3585 as reference.

Reply:
@FraPazGal Thanks for the suggestion, will take hints from there and make changes accordingly.